src/screens/SendTokens.js

```diff
@@ -289,14 +341,19 @@ class SendTokens extends React.Component {
   }
 
   /**
-   * Executed when user clicks to add another token to this transaction  
-   * Checks if still have a known token available that is not selected yet  
+   * Executed when user clicks to add another token to this transaction
+   * Checks if still have a known token available that is not selected yet
    * Create a new child reference with this new token
    */
   addAnotherToken = () => {
     if (hathorLib.wallet.isHardwareWallet()) {
-      $('#notSupported').modal('show');
-      return;
+      if (this.state.txTokens.filter(t => !hathorLib.tokens.isHathorToken(t.uid)).length === LEDGER_TX_CUSTOM_TOKEN_LIMIT) {
+        // limit is 10 custom tokens per tx
+        this.ledgerAlertModalTitle = t`Token limit reached`
+        this.ledgerAlertModalBody = <p>{t`Ledger has a limit of 10 different tokens per transaction.`}</p>
```

Comment:
This message number 10 should use the constant var

Reply:
Changed